src/br/com/zup/nossacasacodigo/author/Author.java

```diff
@@ -1,21 +1,20 @@
 package br.com.zup.nossacasacodigo.author;
 
-import java.util.Date;
-
-import br.com.zup.nossacasacodigo.controllers.AuthorService;
-import br.com.zup.nossacasacodigo.controllers.AuthorsInformarionValidators;
+import java.time.LocalDateTime;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class Author {
 	private String name;
 	private String email;
 	private String description;
-	private Date createdAt;
+	private LocalDateTime createdAt;
 	
-	public Author(String name, String email, String description, Date createdAt) {		
-		AuthorsInformarionValidators.emailValidator(email);
-		AuthorsInformarionValidators.nameValidator(name);
-		AuthorsInformarionValidators.descriptionValidator(description);
-		AuthorsInformarionValidators.dateValidator(createdAt);
+	public Author(String name, String email, String description, LocalDateTime createdAt) {		
```

Comment:
Esse valor sempre vai ser o valor da criação do objeto, nesse caso, por que receber essa data pelo construtor, ao invés de atribuir ao atributo direto, como `this.createdAt = LocalDateTime.now()`?